Files/UserControls/Widgets/DrivesWidget.xaml

```diff
@@ -75,6 +75,23 @@
                             Style="{StaticResource ButtonRevealStyle}"
                             Tag="{x:Bind Path}"
                             ToolTipService.ToolTip="{x:Bind Text, Mode=OneWay}">
+                            <Button.ContextFlyout>
+                                <MenuFlyout>
+                                    <MenuFlyout.Items>
+                                        <MenuFlyoutItem
+                                            x:Name="EjectDevice"
+                                            Tag="{x:Bind Path}"
```

Comment:
We should find a better way to do this so that we don't need to pass in the path.

Reply:
Hi @yaichenbaum I don't find anything particularly wrong with the current approach.
Anyway an alternative approach is in #2347, if you like it more